ReSharper.FSharp/test/src/FSharp.Common.Tests/FSharp.Common.Tests.fsproj

```diff
@@ -15,8 +15,11 @@
   <ItemGroup>
     <PackageReference Include="FSharp.Core" Version="$(FSharpCoreVersion)" />
     <PackageReference Include="JetBrains.FSharp.Compiler.Service" Version="$(FSharpCompilerServiceVersion)" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.0" />
     <PackageReference Include="Moq" Version="4.8.2" />
     <PackageReference Include="NUnit.Console" Version="3.10.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
+    <PackageReference Include="NunitXml.TestLogger" Version="2.1.62" />
```

Comment:
Please make sure it doesn't affect local test runs and internal CI.

Reply:
everything is green there (:

Reply:
everything is green there (: